libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/DialogManager.cs

```diff
@@ -119,63 +123,62 @@ public Dialog RootDialog
         /// Runs dialog system in the context of an ITurnContext.
         /// </summary>
         /// <param name="context">turn context.</param>
-        /// <param name="cancellationToken">cancelation token.</param>
+        /// <param name="cancellationToken">Cancellation token.</param>
         /// <returns>result of the running the logic against the activity.</returns>
-        public async Task<DialogManagerResult> OnTurnAsync(ITurnContext context, CancellationToken cancellationToken = default(CancellationToken))
+        public async Task<DialogManagerResult> OnTurnAsync(ITurnContext context, CancellationToken cancellationToken = default)
         {
             var botStateSet = new BotStateSet();
 
-            // preload turnstate with DM turnstate
-            foreach (var pair in this.TurnState)
```

Comment:
>this [](start = 33, length = 4)

My style guidelines say we SHOULD be using this.

Reply:
this is redundant unless you have ambiguity between a local variable and a field name. We don't use this. in most of the SDK code.

@mrivera-ms & @cleemullins can you guys chime in and clarify what should we use?